configuration/index.md

```diff
@@ -52,146 +56,114 @@ For a recommendation to new users which one(s) to use, see the section below. In
       <td>✔️</td>
-      <td>items/*.items files</td>
+      <td>Main UI</td>
     </tr>
     <tr>
-      <td>Define Groups</td>
-      <td>✔️R</td>
-      <td>✔️</td>
-      <td>❌</td>
+      <td>Define and manage Groups and Items</td>
       <td>✔️</td>
       <td>✔️</td>
+      <td>(✔️)</td>
       <td>items/*.items files</td>
     </tr>
     <tr>
-      <td>Define Sitemaps</td>
-      <td>✔️R</td>
-      <td>❌</td>
-      <td>❌</td>
-      <td>✔️R</td>
-      <td>❌</td>
-      <td>Homebuilder or sitemaps/*.sitemap files</td>
+      <td>Define GUI</td>
+      <td>sitemaps only</td>
+      <td>✔️<br>includes YAML view</td>
+      <td>✔️</td>
+      <td>Main UI<br>sitemaps/*.sitemap files</td>
     </tr>
     <tr>
       <td>Define Transformations</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
-      <td>❌</td>
-      <td>❌</td>
-      <td>transform/*.map files</td>
+      <td>transform/*.map *.js files</td>
     </tr>
     <tr>
       <td>Define Persistence</td>
       <td>✔️</td>
       <td>❌</td>
       <td>❌</td>
-      <td>❌</td>
-      <td>❌</td>
       <td>persistence/*.persist files</td>
     </tr>
     <tr>
       <td>Define Rules</td>
-      <td>✔️R</td>
-      <td>(❌)<br/>only for the experimental rules engine</td>
-      <td>(❌)<br/>not functional</td>
-      <td>❌</td>
+      <td>✔️</td>
+      <td>✔️</td>
       <td>❌</td>
-      <td>rules/*.rules files</td>
+      <td>Main UI<br>for starters: Blockly<br>(graphically create JS code)<br>for advanced users:<br>rules/*.rules files<br>for rules DSL and JSR223</td>
     </tr>
     <tr>
       <td>Manage Z-Wave Devices</td>
       <td>❌</td>
       <td>✔️</td>
-      <td>✔️R</td>
-      <td>❌</td>
       <td>❌</td>
-      <td>HABmin</td>
+      <td>Main UI</td>
     </tr>
     <tr>
       <td>Modify openHAB Settings/Services</td>
       <td>✔️</td>
-      <td>✔️R</td>
       <td>✔️</td>
-      <td>❌</td>
       <td>✔️</td>
-      <td>Paper UI</td>
+      <td>Main UI</td>
     </tr>
     <tr>
       <td>Install Add-ons</td>
-      <td>✔️R<br/>mandatory for OH1 bindings to require config</td>
-      <td>✔️R</td>
       <td>✔️</td>
-      <td>❌</td>
       <td>✔️</td>
-      <td>services/addons.cfg or Paper UI<br/>services/*.cfg for OH1 bindings</td>
+      <td>✔️</td>
+      <td>Main UI</td>
     </tr>
   </tbody>
 </table>
 
 ### Textual vs. Graphical Configuration
 
 In openHAB 1.x, smart home configuration was done via configuration files only.
-openHAB 2 added the administrative web interface "Paper UI".
-It allows for configuration of the base system and setup of openHAB 2 add-ons.
-Please be aware that not every addon/binding of openHAB version 1.x has been updated to version 2 and thus to work with Paper UI.
-You can still run these but you need to use file configuration for them.
+openHAB 2 added the general administrative web interface "Paper UI" and specialized admin UIs such as "habmin" that allow for configuration of the base system and setup of openHAB 2 add-ons.
```

Comment:
Maybe we can remove these sentences complete and just point out that we have a single ui which can also handle the configuration part of openHAB